lib/kubernetes-deploy/errors.rb

```diff
@@ -2,6 +2,7 @@
 module KubernetesDeploy
   class FatalDeploymentError < StandardError; end
   class KubectlError < StandardError; end
+  class InvalidResourceError < StandardError; end
```

Comment:
This should probably use InvalidTemplateError from the PR I just finally rebased and merged. (please rebase 🙏 )